server/test/unit/server/db/test_model.py

```diff
@@ -101,18 +101,39 @@ def test_meta_abstract(self):
 
     @patch('pulp.server.db.model.signals')
     def test_attach_signals(self, mock_signals):
+        model.ContentUnit.attach_signals()
+
+        mock_signals.pre_save.connect.assert_called_once_with(
+            model.ContentUnit.pre_save_signal, sender=model.ContentUnit)
+
+    @patch.dict(model._ContentUnitNamedTupleDescriptor._cache, clear=True)
+    def test_named_tuple(self):
```

Comment:
It looks like this test is actually at least three different unit tests. Each in-line comment seems to describe something that could/should be a stand-alone unit test.